src/python/devcontainer-feature.json

```diff
@@ -22,6 +22,11 @@
       "default": "os-provided",
       "description": "Select a Python version to install."
```

Comment:
Can we bump `minor` version of the package? Thanks!

Reply:
Done in d1f43d335407cdf1c42c2c63dd7eef846894adb0